echo/test/unit/echo_client_test.js

```diff
@@ -237,6 +237,75 @@ describe("Echo Node Client Test Suite", function(){
                 result.body.props.should.be.an.object;
                 result.body.props.url.should.equal("https://foo.bar/baz.html");
 
+                done();
+            });
+        });
+        it("- rounds the timestamp to nearest whole number for arrays of events", function(done){
+            var echoClient = echo.createClient({
+                echo_endpoint: endPoint
+            });
+            
+            var requestStub = sandbox.stub(request, 'post')
+            requestStub.callsFake(function (options, callback) {
+                callback(
+                    null,
+                    {statusCode: 200},
+                    {
+                        "class": "page.views",
+                        "timestamp": 1324524509,
+                        "user": "1234-5678-9012-3456",
+                        "source" : "elevate-app",
+                        "props": {
+                            "url" : "https://foo.bar/baz.html"
+                        }
+                    }
+                );
+            });
+
+            const events = [{class:'class', source:'source', timestamp: 1324524508 }, {class:'class', source:'source', timestamp: 1324524509.123 }];
```

Comment:
suggestion: I wonder if you should include another event with a timestamp of `1324524510.673` and check the result is rounded to the nearest integer (rather than being `floored` for example)?